todo_app/models/index_model.py

```diff
@@ -0,0 +1,25 @@
+from operator import attrgetter
+
+
+class IndexModel:
+    def __init__(self, items):
+        self._items = items
+
+    @property
+    def items(self):
+        return sorted(self._items, key=attrgetter('status'), reverse=True)
```

Comment:
seems an odd way to sort, doing it alphabetically by status? dont think this would result in To do, Doing & Done?

Reply:
This is legacy code, I no longer use it but for when there were only two categories it was useful, I should delete it.